product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/1.1-converting-soap-to-json/1.1.1-soap-to-json-using-payloadfactory-mediator/src/test/resources/testng.xml

```diff
@@ -2,7 +2,7 @@
 <suite name="ESBTestSuite" parallel="false">
     <test name="ProxyService-Test" preserve-order="true" verbose="2" parallel="false">
         <classes>
-            <class name="org.wso2.carbon.esb.endpoint.test.ScenarioSampleTest"/>
+            <!--<class name="org.wso2.carbon.esb.endpoint.test.ScenarioSampleTest"/>-->
```

Comment:
Shall we remove this line as not used?

Reply:
This is fixed by PR [https://github.com/wso2/product-ei/pull/2977](https://github.com/wso2/product-ei/pull/2977)